swift/ql/src/queries/Security/CWE-916/InsufficientHashIterations.qhelp

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+  <overview>
+    <p>Using hash functions with less than 1,000 iterations is not secure. That scheme is vulnerable to password cracking attacks due to having an insufficient level of computational effort.</p>
+  </overview>
+
+  <recommendation>
+    <p>Use sufficient number of iterations (i.e., greater than or equal 1000) for generating password-based keys.</p>
+  </recommendation>
+
+  <example>
+    <p>The following example shows a few cases of instantiating a password-based key. In the 'BAD' cases, the key is initialized with insufficient iterations, making it susceptible to password cracking attacks. In the 'GOOD' cases, the key is initialized with at least 1000 iterations, which protects the encrypted data against recovery.</p>
+    <sample src="InsufficientHashIterations.swift" />
+  </example>
+</qhelp>
```

Comment:
I feel that this `.qhelp` badly needs a `<references>` section, with a link to something that explains why deliberately increasing the compute cost of these functions through iterations is considered a good use of resources (my understanding of the issue here).

Reply:
You're right! I forgot to add them to the initial PR. Added them now.